mlops/bicep/sandbox_fl_confidential.bicep

```diff
@@ -0,0 +1,49 @@
+// This BICEP script will fully provision a federated learning sandbox
```

Comment:
This is the exact same comment as the one in `sandbox_fl_eyesoff_cpu.bicep`. Maybe mention that this one is using CC SKU's by default? (and other differences if applicable)

Reply:
Thank you! fixed